config/environment.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(environment) {
     }, {
       name: 'GoogleTagManager',
       config: {
-        id: process.env.GOOGLE_TAG_MANAGER_ID || (environment === 'production' ? 'GTM-PM94N2' : 'GTM-KJZRH7H')
+        id: process.env.GOOGLE_TAG_MANAGER_ID || (environment === 'production' ? 'GTM-PM94N2' : 'GTM-WNT72NG')
       },
       environments: ['production']
```

Comment:
you'll have to take out this line to get GTM metrics adapter to be included in tests